### PR TITLE
popd: add page

### PR DESCRIPTION
**Pull request by juan88**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by agnivade** [comment]
@katekyan / @juan88 - Please remove the commit for the timeout page and do a force push again. Thanks.

**Comment by juan88** [comment]
Sure no problem. Can you point me in the right direction on how to do it?
It's my second pull request ever so please excuse me if I've made a mistake
while adding my changes.

I also found out that I made an error with the commit message I typed the
wrong command. If you prefer you can dismiss that pull request and I'll do
it again.

2017-10-05 2:40 GMT-03:00 Agniva De Sarker <notifications@github.com>:

**Comment by agnivade** [comment]
So, ideally a single branch should contain commits pertaining to only that PR. In your case, you have the popd branch. This branch has now 4 commits. And we need to get rid of the last commit and push again. The following is what I would do, but there are different ways of achieving this.

In your current branch, do `git reset --hard HEAD^`. This deletes the last commit. And then do a `git push -f origin popd`. That should update the remote branch with the new changes. That's it.

For further assistance, @waldyrious might be able to help you out.

**Comment by juan88** [comment]
That might have happened when I synced the branch with master since another
push had been done right?

I'll try to fix it today and thank you for your assistance.

2017-10-05 12:24 GMT-03:00 Agniva De Sarker <notifications@github.com>:

File: pages/common/popd.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Popd
```
Comment (sbrl) [inline pages/common/popd.md]: `Popd` should be all lowercase here.

File: pages/common/popd.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+> Remove a directory placed on the directory stack via the pushd shell built-in.
+
+- Remove the top directory from the stack and cd to it:
+
+`popd`
+
+- Remove the Nth directory (starting from zero to the left from the list printed with dirs):
```
Comment (sbrl) [inline pages/common/popd.md]: Perhaps enclosing `dirs` in backticks would aid clarity here?

File: pages/common/popd.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`popd +N`
+
+- Remove the Nth directory (starting from zero to the right from the list printed with dirs):
```
Comment (sbrl) [inline pages/common/popd.md]: Same here.

**Comment by juan88** [comment]
Yeah I made a mistake with the title. Been trying to mimic pushd page got
me confused, sorry!

I'll add those backticks and the uppercase fix today.

Thanks for the clarifications!

2017-10-06 9:26 GMT-03:00 Starbeamrainbowlabs <notifications@github.com>:

**Comment by agnivade** [comment]
@juan88 - Please check for the command in osx and windows platforms. If not, duplicate the page into their respective folders.

File: pages/common/popd.md (hunk 3 above)
Comment (juan88) [reply pages/common/popd.md]: Fixed as requested.

**Comment by juan88** [comment]
@agnivade Moved the page to osx and linux since in windows the command takes less parameters and the examples are more appropiate for osx/linux.

The same happens with pushd. In windows, parameters for the command are different. Should be still placed under common?  Want me to open a new issue with this?

https://technet.microsoft.com/en-us/library/bb490969.aspx
https://technet.microsoft.com/en-us/library/bb490978.aspx

**Comment by agnivade** [comment]
Very interesting. In this case, the page should have 3 different copies across the 3 folders. Please add another page under windows, with the windows specific parameters. Thanks for looking into this.

**Comment by juan88** [comment]
@agnivade 
> Please add another page under windows, with the windows specific parameters

Do you mean common folder or should I create a new folder pages/windows for this issue? Thanks!

**Comment by agnivade** [comment]
Oh my bad .. I didn't realise that we don't have a windows folder at all. Yes, please create the new folder and add the page there.

File: pages/linux/popd.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# popd
+
+> Remove a directory placed on the directory stack via the pushd shell built-in.
```
Comment (agnivade) [inline pages/linux/popd.md]: "via the pushd shell built-in" => "by the `pushd` command" 
To make it consistent with the windows page. And also command sounds better than shell built-in. :laughing:

**Comment by agnivade** [comment]
@juan88 - Thanks for being patient with the review process. We are nearly there. Just one small thing to take care of.

**Comment by sbrl** [comment]
The new windows folder begins! If I end up using a windows machine in the near future and I remember, I'll look at sending a PR for a few windows-specific commands :P

**Comment by juan88** [comment]
Sure! No problem! Always glad to help and learn! :+1:

**Comment by sbrl** [comment]
Thanks, @juan88! Thanks for sticking with it throughout the review process :smiley_cat:

**Pull request merged**